pkg/client/requestaccess.go

```diff
@@ -0,0 +1,59 @@
+package client
+
+import (
+	"fmt"
+
+	"github.com/storacha/go-libstoracha/capabilities/access"
+	uclient "github.com/storacha/go-ucanto/client"
+	"github.com/storacha/go-ucanto/core/invocation"
+	"github.com/storacha/go-ucanto/did"
+	"github.com/storacha/go-ucanto/principal"
+)
+
+// SpaceAccess is the set of capabilities required by the agent to manage a
+// space.
+var SpaceAccess = []string{
+	"space/*",
+	"blob/*",
+	"index/*",
+	"store/*",
+	"upload/*",
+	"access/*",
+	"filecoin/*",
+	"usage/*",
+}
```

Comment:
what about defining this directly as a slice of `access.CapabilityRequest`?
```suggestion
var SpaceAccess = []access.CapabilityRequest{
	{Can: "space/*"},
	{Can: "blob/*"},
	{Can: "index/*"},
	{Can: "store/*"},
	{Can: "upload/*"},
	{Can: "access/*"},
	{Can: "filecoin/*"},
	{Can: "usage/*"},
}
```
that way you can avoid converting it each time afterwards. Not as clean maybe, but still straightforward enough.

Reply:
Yeah, I like that. This was me over-literally translating JS. 😄